ui/apps/platform/src/Containers/NetworkGraph/components/StyleGroup.tsx

```diff
@@ -77,12 +77,21 @@ const StyleGroup: React.FunctionComponent<StyleGroupProps> = ({
         return newData as CustomGroupNodeData;
     }, [data]);
 
+    let className = '';
+
+    if (passedData.type === 'NAMESPACE') {
+        className = `${className} ${
+            passedData.isFilteredNamespace ? 'filtered-namespace' : 'derived-namespace'
+        }`.trim();
+    }
+
```

Comment:
We do this class name interpolation fairly frequently throughout the app. 

I'm not one to advocate for 3rd-party libraries much, but the `class-names` package would clean up all these interpolations.

https://github.com/JedWatson/classnames#readme

Reply:
I think that's a decent suggestion. Maybe we can look into it to see if it's worth adding. We can also see if we can make our own solution. Maybe we can consider this after we start adding a bit more classes. That way we can have more examples to go off of.